src/main/java/step3/domain/ConfirmResults.java

```diff
@@ -0,0 +1,66 @@
+package step3.domain;
+
+import step3.constants.PrizeGrade;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+public class ConfirmResults {
+
+	private final Map<PrizeGrade, Integer> groupedByPrizeGrade;
+
+	private static final int ZERO = 0;
+	private static final int INCREASE = 1;
+
+	public ConfirmResults(List<PrizeGrade> prizeGrades) {
+		this.groupedByPrizeGrade = getGroupByPrizeGrade(prizeGrades);
+	}
+
+	public Map<PrizeGrade, Integer> getGroupedByPrizeGrade() {
+		return groupedByPrizeGrade;
+	}
```

Comment:
Collections.unmodifiableMap()를 활용하면 read-only로 리턴하여 외부에서 컬렉션의 변조를 막을 수 있습니다.
```suggestion
	public Map<PrizeGrade, Integer> getGroupedByPrizeGrade() {
		return Collections.unmodifiableMap(groupedByPrizeGrade);
	}
```